test/integration/full/configure-options/configure-options.js

```diff
@@ -0,0 +1,50 @@
+describe('Check Configure Options', function() {
+	'use strict';
+
+	var target = document.querySelector('#target');
+
+	describe('aria-allowed-attr', function() {
+		it('should allow an attribute supplied in options', function(done) {
+			target.setAttribute('role', 'separator');
+			target.setAttribute('aria-valuenow', '0');
```

Comment:
In the event that 'aria-valuenow' becomes an allowed attribute in our lookup table, this test won't be testing anything anymore. Just to make the test more future-proof, I suggest using a random value like 'peanutbutter' instead of a real attribute so we can guarantee the test will always be testing the override functionality with a truly unexpected value.